arch/aarch64/mcount-support.c

```diff
@@ -114,6 +114,7 @@ void mcount_get_stack_arg(struct mcount_arg_context *ctx,
 			  struct uftrace_arg_spec *spec)
 {
 	int offset = 1;
+	unsigned long *addr = ctx->stack_base;
```

Comment:
Why not adding offset here?

Reply:
Oh, I forgot the code.. sorry for the noise. ;-p

Looking at the code, it looks the initial value of 1 is just for documentation purpose.  It should be greater than or equal to 1.  And the actual value will be calculated from the argspec type anyway.